### PR TITLE
Delete requirements.txt.

### PR DESCRIPTION
**Pull request by LiterallyUniqueLogin**
Also Clarify bioconda build process in PUBLISHING.rst

### CODE REVIEW

File: PUBLISHING.rst [hunk 1]
```diff
@@ -51,15 +64,25 @@ Then go through the steps of publishing the changed code to PyPI:
 
 4. Run :code:`twine upload dist/*` to upload the build to PyPI
 
-Lastly go through the following steps to publish the changed code to bioconda: (see `here <http://bioconda.github.io/contributor/workflow.html>`_ for official documentation)
+Lastly, the change needs to be published to bioconda.
+
+A bioconda bot will automatically open a pull request (within a day?) updating the version number
+and the PyPI reference. If there are no new dependencies, no changes the to the build,
```
Comment (aryarm) [inline PUBLISHING.rst:70]:
```suggestion
and the PyPI reference. If there are no new dependencies, no changes to the build,
```

File: PUBLISHING.rst [hunk 2]
```diff
@@ -6,27 +6,40 @@ If you are a community member and want to contribute new code, see the contribut
...
 We use the master branch as the default branch with the latest stable codebase.
 The builds from this branch are distributed to PyPI and conda.
 The develop branch contains new features that have yet to make their way into master.
 
+New Dependencies
+----------------
+If you've added dependencies to trtools or its tests, those dependencies should be listed in
+
+  * setup.py
+  * the .readthedocs_conda_env.yml file in the root of the repositiory that's used for building
```
Comment (aryarm) [inline PUBLISHING.rst:24]:
```suggestion
  * the .readthedocs_conda_env.yml file in the root of the repository that's used for building
```

**Review by aryarm — APPROVED** [review]
looks good, @LiterallyUniqueLogin ! I don't see any problems with the tweaked workflow

This might be out of the scope of this PR, but one thing I noticed is that PUBLISHING.rst doesn't have any instructions on how to build the documentation locally, which might help people iterate and preview changes to the docs faster. Might be something to consider for a future PR or something.

**Comment by LiterallyUniqueLogin** [comment]
> looks good, @LiterallyUniqueLogin ! I don't see any problems with the tweaked workflow
> 
> This might be out of the scope of this PR, but one thing I noticed is that PUBLISHING.rst doesn't have any instructions on how to build the documentation locally, which might help people iterate and preview changes to the docs faster. Might be something to consider for a future PR or something.

That info is in the contributing section of the README . If you have a spot you think it should go in PUBLISHING also, let me know and I'm happy to add it there.

**Comment by aryarm** [comment]
ah, I see it now! ok, looks great

no, I don't think it needs to also be in PUBLISHING in that case

**Pull request merged**